SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+		} elseif ($esxiConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			$esxiConfigTypeValues.$($configuration.Name) = $configuration.Value
+		} else {
+			Write-LogMessage -Type ERROR -Message "parameter $($configuration.Name) not found (ESXi)" -Colour Red
+		}
+	}
+	foreach ($configuration in $productConfiguration.Default.PSObject.Properties) {
+		if ($esxiConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			if ($esxiConfigTypeValues.$($configuration.Name) -eq "Default") {
+				$esxiConfigTypeValues.$($configuration.Name) = $configuration.Value
+			}
+		}
+		if ($configuration.Name -eq "passwdMinimumLength") {
+			if ($esxiConfigTypeValues.passwdMinimumLengthFor4CharClass -eq "Default") {
+				$esxiConfigTypeValues.passwdMinimumLengthFor4CharClass = $configuration.Value
+			}
+		}
+	}
+	$standardConfigValue = New-Object -TypeName psobject
+	$standardConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "ESXi"
+	$standardConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue $esxiConfigTypeValues
+	$ppmStandardConfigValues.Add($standardConfigValue)
+}
+Function parseProductTypeVCConfiguration{
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$productConfiguration,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues
+	)
+	
+	$VCConfigTypeValues =  New-Object -TypeName psobject
+	$VCConfigTypeValues | Add-Member -notepropertyname "passwdExpInDays" -notepropertyvalue "Default" 
+	$VCConfigTypeValues | Add-Member -notepropertyname "passwdNotifyEmail" -notepropertyvalue "Default"
+	foreach ($configuration in $productConfiguration.VC.PSObject.Properties) {
+		if ($VCConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			$VCConfigTypeValues.$($configuration.Name) = $configuration.Value
+		} else {
+			 Write-LogMessage -Type ERROR -Message "parameter ("$configuration.Name") not found (VC)" -Colour Red
+		}
+	}
+	foreach ($configuration in $productConfiguration.Default.PSObject.Properties) {
+		if ($VCConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			if ($VCConfigTypeValues.$($configuration.Name) -eq "Default") {
+				$VCConfigTypeValues.$($configuration.Name) = $configuration.Value
+			}
+		}
+	}
+	$standardConfigValue = New-Object -TypeName psobject
+	$standardConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "VC"
+	$standardConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue $VCConfigTypeValues
+	$ppmStandardConfigValues.Add($standardConfigValue)
+}
+Function parseProductTypeSSOConfiguration{
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$productConfiguration,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues
+	)
```

Comment:
Add blank line after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175